lib/galaxy/datatypes/media.py

```diff
@@ -198,30 +287,14 @@ class Mp4(Video):
     file_ext = "mp4"
 
     def sniff(self, filename: str) -> bool:
-        if which("ffprobe"):
-            metadata, streams = ffprobe(filename)
-            return "mp4" in metadata["format_name"].split(",")
-        return False
+        return _get_file_format_from_magic_number(filename, "mp4")
 
 
 class Flv(Video):
     file_ext = "flv"
 
     def sniff(self, filename: str) -> bool:
-        if which("ffprobe"):
-            metadata, streams = ffprobe(filename)
-            return "flv" in metadata["format_name"].split(",")
-        return False
-
-
-class Mpg(Video):
```

Comment:
this was replaced by Mpeg? Removing a datatype will affect existing users of this type, as well as any tools that expect that type to exist, I don't know if that's a logical choice. Renaming the file extension, sure, that'd be fine. (datatypes_conf.xml will also need to be corrected)

Reply:
MPG and MPEG files are pretty much the same thing. Usually, the steam is MPEG for both of them, but let's stick with MPG since changing it causes problems.